source/features/quick-comment-edit.tsx

```diff
@@ -71,7 +93,11 @@ async function init(signal: AbortSignal): Promise<void> {
 	// If true then the resulting selector will match all comments, otherwise it will only match those made by you
 	const preSelector = canEditEveryComment() ? '' : '.current-user';
 
-	observe(preSelector + '.js-comment.unminimized-comment .timeline-comment-actions details.position-relative', addQuickEditButton, {signal});
+	observe([
+		'button[aria-label="Issue body actions"]',
+		preSelector + '.js-comment.unminimized-comment .timeline-comment-actions details.position-relative', // TODO: Drop in March 2024
```

Comment:
March 2024?

Reply:
We drop old code about 6 months after the update to give a chance to GH Enterprise users to get the new UI. Not a guarantee, but if it's easy we keep both versions

Reply:
```suggestion
		preSelector + '.js-comment.unminimized-comment .timeline-comment-actions details.position-relative', // TODO: Drop in March 2025
```